docs/creating_execs_and_libs.rst

```diff
@@ -106,3 +115,8 @@ without the linker removing unused symbols in the larger library.
     blt_add_exectuble(NAME       helloWorld
                       SOURCES    main.cpp
                       DEPENDS_ON myObjectLibrary)
+
+.. note::
+  Due to record keeping on BLT's part to make object libraries as easy to use
+  as possible, you need to define object libraries before you use them
```

Comment:
Just to make sure that I understand this -- it sounds like typical CMake targets can be used before they are defined, but that is not the case for object libraries. Is that right?

Reply:
Yes. I find it very strange but you can.